.github/workflows/ci.yml

```diff
@@ -17,15 +17,15 @@ jobs:
         uses: actions/setup-node@v3
         with:
           node-version: ${{ matrix.node }}
-      - run: npm install
+      - run: npm ci
       - run: npm run build
 
   lint:
     runs-on: ubuntu-latest
 
     strategy:
       matrix:
-        node: [14, 16]
+        node: [16]
```

Comment:
I don't think it's toooo useful to lint in many versions. We can discuss to add 18 here